### DISCUSSION TITLE
Curious if any testing has been done with Starlink

### DISCUSSION THREAD

**Discussion by oviano** [Haivision/srt#2991] [General]
I was reading this article earlier and for no particular reason I found myself wondering how SRT would behave with a Starlink connection.

https://labs.apnic.net/index.php/2024/05/16/a-transport-protocols-view-of-starlink/#:~:text=The%20third%20is%20that%20there,latency%20variation%20is%20relatively%20high.

**Comment by timokorkalainen** [comment]
I have done some testing and so far seems a bit hit and miss. Some connections have been perfect for hours, others have barely even got a single frame through. Too many variables to call anything certain though.

**Reply by Satellite-Bruce** [reply]
A fair bit. The Starlink is on a ship. I'm using HEVC at 3 Mbs. Latency is usually stable around 200 ms RTT. It is not perfect but better than you would expect. This is also not a basic contract with Starlink.
